src/Components/GEOSldas_GridComp/GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -317,84 +331,88 @@ subroutine get_forcing( date_time, force_dtstep, met_path, met_tag,        &
-       ! are needed in subroutine interpolate_to_timestep(), and
-       ! "met_force_obs_tile_new" is set to nodata for forcing fluxes.
-       
-       ! The calls below are moved to LDAS_move_new_force_to_old
-       
-       ! Note also "bkwd_looking_fluxes" switch. - reichle, 22 Apr 2021
-       
-       ! met_force_obs_tile_old%Rainf_C = met_force_obs_tile_new%Rainf_C
-       ! met_force_obs_tile_old%Rainf   = met_force_obs_tile_new%Rainf
-       ! met_force_obs_tile_old%Snowf   = met_force_obs_tile_new%Snowf
-       ! met_force_obs_tile_old%LWdown  = met_force_obs_tile_new%LWdown
-       ! met_force_obs_tile_old%SWdown  = met_force_obs_tile_new%SWdown
-       ! met_force_obs_tile_old%SWnet   = met_force_obs_tile_new%SWnet
-       ! met_force_obs_tile_old%PARdrct = met_force_obs_tile_new%PARdrct
-       ! met_force_obs_tile_old%PARdffs = met_force_obs_tile_new%PARdffs
-       
-       ! treat Wind as flux when forcing with MERRA
-       
-       ! if (MERRA_file_specs) met_force_obs_tile_old%Wind  = met_force_obs_tile_new%Wind
-       
-       ! met_force_obs_tile_new%Rainf_C = nodata_generic
-       ! met_force_obs_tile_new%Rainf   = nodata_generic
-       ! met_force_obs_tile_new%Snowf   = nodata_generic
-       ! met_force_obs_tile_new%LWdown  = nodata_generic
-       ! met_force_obs_tile_new%SWdown  = nodata_generic
-       ! met_force_obs_tile_new%SWnet   = nodata_generic
-       ! met_force_obs_tile_new%PARdrct = nodata_generic
-       ! met_force_obs_tile_new%PARdffs = nodata_generic
+    end if
 
-       ! if (MERRA_file_specs) met_force_obs_tile_new%Wind = nodata_generic
-       
+    ! ------------------
+    !
+    ! backfill PAR:
+    !
+    ! PAR might be missing for one of the following reasons:
+    ! - the reader did not provide PAR
+    ! - the reader provides PAR but owing to a land mask difference between
+    !     the forcing data and the LDAS simulation, some tiles may not have PAR 
+    ! - the reader provides PAR but there are gaps in the original dataset
+    !
+    ! if PAR was not available at all from the forcing reader, assign "nodata_forcing"
+
+    if (.not. PAR_available) then
+       met_force_obs_tile_new%PARdrct = nodata_forcing
+       met_force_obs_tile_new%PARdffs = nodata_forcing
     end if
+
+    ! the nodata-value for all forcing fields (except SWnet) is now "nodata_forcing"
+    
+    ! where PAR is not available for any of the above reasons:
+    ! - assume half of SWdown is photosynthetically active
+    ! - assume half of PAR is direct, half diffuse
     
-    ! check for nodata values, possibly fill with neighboring data
+    tol = abs(nodata_forcing*nodata_tolfrac_generic)
+    
+    where (                                                                       &
+         (abs(met_force_obs_tile_new%PARdrct-nodata_forcing) < tol)  .and.        &
+         (abs(met_force_obs_tile_new%SWdown -nodata_forcing) < tol)            )
```

Comment:
I think this is incorrect.   If both are true, then there is no data

Reply:
Good catch, thanks!  SWdown must NOT be no-data.  I'll fix it.